subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/video/VideoCaptureTestListener.kt

```diff
@@ -63,23 +71,24 @@ class VideoCaptureTestListener(
     override fun afterTestStop(state: ReportState.Initialized.Started) {
         if (videoFeature.videoUploadingEnabled(shouldRecord, savedIncident)) {
             logger.debug("Video uploading enabled. Recording stopping...")
-            val videoFile = videoCapturer.stop()
-            logger.debug("Video uploading enabled. Recording stopped")
-
-            if (videoFile != null) {
-                val video = remoteStorage.upload(
-                    uploadRequest = RemoteStorage.Request.FileRequest.Video(
-                        file = videoFile
-                    ),
-                    comment = "video"
-                )
-                logger.debug("Video uploading enabled. Video uploaded")
-
-                waitUploads(state = state, video = video)
-            } else {
-                logger.debug(
-                    "Video uploading enabled. Filed to upload video for ${state.testMetadata.className}.${state.testMetadata.methodName}"
-                )
+            when (val result = videoCapturer.stop()) {
```

Comment:
Before we could fail in `stop` and it led to failure in `com.avito.android.test.report.ReportImplementation#reportTestCase` because of that we didn't call `com.avito.android.test.report.ReportImplementation#writeTestCase` where we write test result